src/e2e/puppeteer/__tests__/multiselect.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'path'
+import { KnownDevices } from 'puppeteer'
+import configureSnapshots from '../configureSnapshots'
+import emulate from '../helpers/emulate'
+import longPressThought from '../helpers/longPressThought'
+import multiselectThoughts from '../helpers/multiselectThoughts'
+import paste from '../helpers/paste'
+import screenshot from '../helpers/screenshot'
+
+expect.extend({
+  toMatchImageSnapshot: configureSnapshots({ fileName: path.basename(__filename).replace('.ts', '') }),
+})
+
+describe('multiselect', () => {
+  it('should multiselect two thoughts at once', async () => {
+    await paste(`
+        - a
+        - b
+        `)
+
+    await multiselectThoughts(['a', 'b'])
+
+    expect(await screenshot()).toMatchImageSnapshot({ customSnapshotIdentifier: 'multiselect' })
```

Comment:
Let's use [hideHUD](https://github.com/cybersemics/em/blob/7a51ebcb9c7263a0f306016f56d08fedf56c6c0a/src/e2e/puppeteer/helpers/hideHUD.ts) to avoid accidentally tightly coupling the test to the toolbar rendering.